src/components/Sidebar/TrackKeys/TrackKey.jsx

```diff
@@ -22,7 +22,8 @@ const TrackKey = ({ track, toggleOpen }, context) => {
           </button>
         }
         <span>{title}</span>
-        { hasButton && clickTrackButton && <button className="rt-track-key__button" onClick={handleClick} /> }
+        { hasButton && !sideComponent && clickTrackButton && <button className="rt-track-key__button" onClick={handleClick} /> }
```

Comment:
Also, we should re-use "side" to display the button too i.e. just a specialisation